src/Statement/StatementList.php

```diff
@@ -50,7 +50,7 @@ public function __construct( $statements = array() /*...*/ ) {
 
 	/**
 	 * @since 3.0
-	 * 
```

Comment:
I fixed this directly in master as this change does not need any discussion and makes all the tests fail.